cmd/kubeless/trigger/http/create.go

```diff
@@ -88,6 +88,15 @@ var createCmd = &cobra.Command{
 		}
 		httpTrigger.Spec.TLSAcme = enableTLSAcme
 
+		tlsSecret, err := cmd.Flags().GetString("tls-secret")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		if enableTLSAcme && len(tlsSecret) > 0 {
+			logrus.Fatalf("Either --enableTLSAcme or --tls-secret must be specified ")
```

Comment:
this error is a bit misleading. I would suggest `Cannot specify both --enableTLSAcme and --tls-secret`